frontend/dispatcher.lua

```diff
@@ -109,6 +109,7 @@ local settingsList = {
     folder_shortcuts = {category="none", event="ShowFolderShortcutsDialog", title=_("Folder shortcuts"), filemanager=true, separator=true},
 
     -- reader settings
+    open_next_document_in_folder = {category="none", event="OpenNextDocInFolder", title=_("Open next document in folder"), reader=true, separator=true},
```

Comment:
No real reason to abbreviate "Doc" in the event name, as the text and the key here have "document" in full.
`OpenNextDocInFolder` => `ReaderStatus:onOpenNextDocumentInFolder`